app/src/main/java/org/mozilla/fenix/home/sessions/CurrentSessionBottomSheetFragment.kt

```diff
@@ -36,7 +36,9 @@ class CurrentSessionBottomSheetFragment : BottomSheetDialogFragment(), LayoutCon
         }
 
         view.delete_session_button.setOnClickListener {
-            requireComponents.core.sessionManager.removeAll()
+            sessions.forEach {
+                requireComponents.core.sessionManager.remove(it)
+            }
```

Comment:
Aye. We should add a helper for that to AC. I'm going to file something and write a patch. :)

`removeAll()` was definitely problematic since it really removes *all* sessions including custom tabs. There's `removeSessions()` to just remove the regular sessions - but that doesn't have the ability to do that for private tabs only.

Reply:
Thanks, Sebastian! I agree this could use a solution from A-C.